docs/readme.md

```diff
@@ -0,0 +1,109 @@
+Useful documentation, examples, and [recipes](./recipes/) to get you started.
```

Comment:
I actually for that very reason :)...so people would see some of the most common examples when they hit /docs. It's not really a table of contents though. `[recipes](./recipes/)` will give them the list of directories/md files.